react/icons/CheckMarkIcon/CheckMarkIcon.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="90.471 302.809 823 823"><g></g><g> <g><polygon points="365.407,1022.185 90.677,747.457 158.56,679.575 365.406,886.421 845.383,406.432 913.266,474.314"/></g></g> </svg>
```

Comment:
Has this asset been optimised? Seems to have some redundant group elements. @stephaniemoss has previously tidied up assets for me before adding. Not a big deal, just noticed it